config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatasets.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   creationTimestamp: null
   labels:
     cnrm.cloud.google.com/managed-by-kcc: "true"
-    cnrm.cloud.google.com/stability-level: stable
+    cnrm.cloud.google.com/stability-level: alpha
```

Comment:
This diff means you need to add the stability-level tag in the API. Here's an example https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8fe8c6dd64cc1034dfb2ab00105bf258dc885406/apis/kms/v1beta1/kmskeyring.go#L73